lang/src/org/partiql/lang/ast/ExprNodeToStatement.kt

```diff
@@ -48,6 +49,18 @@ private fun ExprNode.toAstDdl(): PartiqlAst.Statement {
     }
 }
 
+private fun ExprNode.toAstExec() : PartiqlAst.Statement {
+    val node = this
+    val metas = metas.toElectrolyteMetaContainer()
+
+    return PartiqlAst.build {
+        when (node) {
+            is Exec -> exec(node.procedureName.name, node.args.map { it.toAstExpr() }, metas)
```

Comment:
`procedureName.meta` is not kept in `PartiqlAst.Statement`? Not to say it's important but just want to confirm.

Reply:
That particular `meta` may not be too important, but we should be copying metas wherever possible. I will add that in the next commit.

I noticed there are quite a few other metas that aren't copied when converting `ExprNode`<-> `PartiqlAst`, so I created an issue (#353) to track.